TankDriver.App/Models/BulletSpaceModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using TankDriver.Logic;
+
+namespace TankDriver.Models
+{
+	internal class BulletSpaceModel: IModel
+	{
+		private BulletSpace _bulletSpace;
+		private TextureStorage _textureStorage;
+
+		public BulletSpaceModel (BulletSpace bulletSpace)
+		{
+			_bulletSpace = bulletSpace;
+		}
+
+		public void LoadBulletTextures(IModel bulletModel)
+		{
+			bulletModel.LoadTextures (_textureStorage);
+		}
+
+		#region IModel implementation
+
+		public IUnit GetUnit ()
+		{
+			return _bulletSpace;
+		}
+
+		public void LoadTextures (TextureStorage textureStorage)
+		{
+			_textureStorage = textureStorage;
+		}
+
+		public void Render (SpriteBatch spriteBatch)
+		{
+			foreach (var bullet in _bulletSpace.Bullets) {
+				bullet.GetModel ().Render (spriteBatch);
+			}
+		}
+
+		#endregion
+	}
+}
+
```

Comment:
There's no need to separate `BulletSpace` and `BulletSpaceModel`, so yeah, merge them.

And I don't like the idea that we want the separate space for every kind of stuff, but as I said, I'll think about that later.